### ISSUE TITLE
'missing_grant_type' error when using exchangeOAuthCodeForAccessToken in TypeScript

### ISSUE THREAD

**Issue by jeremiese** [FusionAuth/fusionauth-typescript-client#21]
When using the TypeScript library, I always get the 'missing_grant_type' error when I use `exchangeOAuthCodeForAccessToken`. 

My setup:

FusionAuth 1.16 (Docker)
TypeScript FusionAuth Client: 1.16.1
NodeJS 12.16.3

The same function works flawlessly with the NodeJS FusionAuth client. I looked at the code for the TypeScript version, and the code says "body.append("grant_type", "authorization_code")", so I'm wondering why it fails to recognize it.

Please let me know if you need further information.

**Comment by mooreds** [comment]
Hmm. I ran into the same issue here: https://github.com/FusionAuth/fusionauth-typescript-client/issues/20 but upgrading to 1.16.1 fixed it. Here's my sample code that works: https://github.com/FusionAuth/fusionauth-example-node

Can you try the example code against your FusionAuth server and see if that works?

**Comment by jeremiese** [comment]
Thanks a lot! My version of FusionAuth is 1.16 (Docker) and my TypeScript client version is 1.16.1 (I just upgraded yesterday and tried, and it still failed). 

The difference I see with your setup is that you're in pure Node with the TypeScript FusionAuth client whereas I'm using TypeScript for my whole code plus the TypeScript FusionAuth client. So far, I've had success using TypeScript + NodeJS FusionAuth client, but NOT TypeScript + TypeScript FusionAuth client. 

So maybe something's out of order with my TypeScript setup. I just wanted to move to the TypeScript client since FusionAuth indicates on their site that the Node version is deprecated in favor of the TypeScript one.

**Comment by jeremiese** [comment]
I went back to my code and checked deeper, and you're right, it was only the 1.16.1. I had upgraded to 1.16.1 in another closely related service, so I mixed them both and thought I had upgraded in service 1 while the upgrade was only in service 2. 

So I made sure both my services were upgraded to 1.16.1, and now everything works perfectly fine using TypeScript + TypeScript FusionAuth client. 

Thanks a lot for you help! That was an easy fix.
